trustee-on-GCP/rh-coreos/Containerfile

```diff
@@ -0,0 +1,18 @@
+FROM quay.io/rkaufman/kbs-tpm-snp:latest as kbc
+FROM quay.io/openshift-release-dev/ocp-v4.0-art-dev:419.96.202505021444-0-coreos
+
+COPY usr /usr
```

Comment:
You can copy directory by directory based on the argument/label. We should really maintain a single Containerfile